docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -68,7 +73,16 @@ Folders help you organize and group dashboards, which is useful when you have ma
 1. On the **Dashboards** page, click **New** and select **New folder** in the drop-down.
 1. Enter a unique name and click **Create**.
 
-When you save a dashboard, you can either select a folder for the dashboard to be saved in or create a new folder.
+When you save a dashboard, you can optionally select a folder to save the dashboard in.
+
+You can also create a folder within an existing folder:
+
+1. Click **Dashboards** in the main menu.
+1. On the **Dashboards** page, click an existing folder name to navigate to that folder.
+1. On the folder's page, click **New** and select **New folder** in the drop-down.
+1. Enter a unique name and click **Create**.
+
+You can nest folders up to four levels deep.
```

Comment:
Can this just be an optional step within the create a dashboard folder task? Kind of like:

1. Click Dashboards in the main menu.
2. Do one of the following:
  - On the Dashboards page, click New and select New folder in the drop-down.
  - Click an existing folder and on the folder’s page, click New and select New folder in the drop-down.
 3. Enter a unique name and click Create.

When you nest folders, you can do so up to four levels deep.